apps/website-new/docs/en/guide/basic/rspack.mdx

```diff
@@ -32,34 +32,36 @@ import { PackageManagerTabs } from '@theme';
 In `Rspack`, you can add the plugin through the `plugins` configuration item in the `rspack.config.js` file:
 
 ```ts title='rspack.config.js'
+import { defineConfig } from '@rsbuild/core';
 import { ModuleFederationPlugin } from '@module-federation/enhanced/rspack';
-module.exports = {
-  devServer: {
-    port: 2000,
+
+export default defineConfig({
+  // publicPath must be configured if using manifest
+  dev: {
+    assetPrefix: 'http://localhost:2000/',
   },
   output: {
-    // publicPath must be configured if using manifest
-    publicPath: 'http://localhost:2000/',
+    assetPrefix: 'https://cdn.domain.com/path/to/project/',
```

Comment:
It would be nice to add another comment to this configuration

Reply:
For example, this will affect the build product resource prefix in production mode